account_financial_report/models/account_age_report_configuration.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2023 Ernesto García
+# Copyright 2023 Carolina Fernandez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class AccountAgeReportConfiguration(models.Model):
+    _name = "account.age.report.configuration"
+    _description = "Model to set intervals for Age partner balance report"
+
+    name = fields.Char(required=True)
+    company_id = fields.Many2one(
+        "res.company", default=lambda self: self.env.company, readonly=True
+    )
+    line_ids = fields.One2many(
+        "account.age.report.configuration.line", "account_age_report_config_id"
+    )
+
+    @api.constrains("line_ids")
+    def _check_line_ids(self):
+        for rec in self:
+            if not rec.line_ids:
+                raise ValidationError(_("Must complete Configuration Lines"))
+
+
+class AccountAgeReportConfigurationLine(models.Model):
+    _name = "account.age.report.configuration.line"
+    _description = "Model to set interval lines for Age partner balance report"
+
+    name = fields.Char(required=True)
+    code = fields.Char(required=True)
+    account_age_report_config_id = fields.Many2one("account.age.report.configuration")
+    lower_limit = fields.Integer()
```

Comment:
We talk here about putting only one value, so no need to check overlappings and missing gaps:

Configuration 1:
- 15
- 30
- 60

It means the first interval is from 0 to 15, the second from 16 to 30, and the third is  61+.

Reply:
Done